From: "Andrew F. Davis" <afd@ti.com>
Date: Fri, 31 Aug 2018 14:13:22 -0500
Subject: gpio: davinci: Remove unused member of davinci_gpio_controller
Git-commit: 329357723f5e8e2d7fc3144fb79936b51ce63b76
Patch-mainline: v4.20-rc1
References: git-fixes

This was added as part of the patch in the fixes below, but was
not needed or used, remove this here.

Fixes: 8e11047b8f3c ("gpio: davinci: Add support for multiple GPIO controllers")
Tested-by: Keerthy <j-keerthy@ti.com>
Acked-by: Keerthy <j-keerthy@ti.com>
Signed-off-by: Andrew F. Davis <afd@ti.com>
Signed-off-by: Linus Walleij <linus.walleij@linaro.org>
Acked-by: Nicolas Saenz Julienne <nsaenzjulienne@suse.de>
---
 include/linux/platform_data/gpio-davinci.h |    1 -
 1 file changed, 1 deletion(-)

--- a/include/linux/platform_data/gpio-davinci.h
+++ b/include/linux/platform_data/gpio-davinci.h
@@ -42,7 +42,6 @@ struct davinci_gpio_controller {
 	void __iomem		*regs[MAX_REGS_BANKS];
 	int			gpio_unbanked;
 	unsigned int		base_irq;
-	unsigned int		base;
 };
 
 /*
